docs/guides/core-concepts/retry-ability.mdx

```diff
@@ -29,25 +29,30 @@ fails, check out our guide on [Test Retries](/guides/guides/test-retries).
 
 ## Commands, Queries and Assertions
 
-While all methods you chain off of `cy` in your Cypress tests are commands,
-there are some different types of commands it's important to understand:
-**queries**, **assertions** and **actions** have special rules about
-retryability. For example, there are 4 queries, an action, and 2 assertions in
-the test below.
+While all methods you chain off of `cy` in your Cypress tests are commands, it's
+important to understand the different rules by which they operate. **Queries**
+link up, retrying the entire chain together, while **non-queries** only execute
+once. Assertions are a type of query that's specially displayed in the command
```

Comment:
Nit: bullet point each type to make this easier to read. 
```suggestion
once. **Assertions** are a unique query such that it ensures an explicit condition is met to continue executing the Command Queue. 
```

Reply:
Bullet-pointed the types.